packages/cli/src/Generate.ts

```diff
@@ -215,8 +215,10 @@ Please run \`${getCommandWithExecutor('prisma generate')}\` to see the errors.`)
           clientVersionBeforeGenerate &&
           typeof clientVersionBeforeGenerate === 'string'
         ) {
+          const major = clientVersionBeforeGenerate.split('.')[0]
```

Comment:
```ts
const [major, minor] = clientVersionBeforeGenerate.split('.')
```

Reply:
No, this was just the Engineering Manager writing code - @millsp fixed it properly.